OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -126,12 +126,18 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 				var type = info.RevealGeneratedShroud ? Shroud.SourceType.Visibility
 					: Shroud.SourceType.PassiveVisibility;
 
+				var revealDuration = info.SkipAscent ? (info.FlightDelay / 2) - info.CameraSpawnAdvance
```

Comment:
These can be shortened down to `info.FlightDelay / (info.SkipAscent ? 2 : 1) - info.CameraSpawnAdvance;`